numpy/lib/function_base.py

```diff
@@ -1501,14 +1505,20 @@ def unwrap(p, discont=pi, axis=-1):
     array([ 0.        ,  0.78539816,  1.57079633, -0.78539816,  0.        ]) # may vary
 
     """
+    if min_val is None:
+        min_val = -pi
+    if max_val is None:
+        max_val = pi
```

Comment:
Why not just put these defaults in the signature?

Reply:
My initial setup had left them out since I wanted to know if they were explicitly specified but that is only necessary for `discont`